frontend/projects/upgrade/src/app/features/dashboard/home/components/metrics/metrics.component.ts

```diff
@@ -98,7 +112,7 @@ export class MonitoredMetricsComponent implements OnInit, OnChanges, OnDestroy {
           operationType: [null, Validators.required],
           compareFn: [null, Validators.required],
           compareValue: [null, Validators.required],
-          repeatedMeasure: [REPEATED_MEASURE.mostRecent, Validators.required],
+          repeatedMeasure: [null, Validators.required],
```

Comment:
It looks like this line is behind @zackcl 's problem. Should repeated measure have a default value if it's not allowed to be null?

Reply:
Actually no, it's line 289